workbench_utils.py

```diff
@@ -5614,7 +5614,7 @@ def create_children_from_directory(config, parent_csv_record, parent_node_id):
         # Execute node-specific post-create scripts, if any are configured.
         if 'node_post_create' in config and len(config['node_post_create']) > 0:
             for command in config['node_post_create']:
-                post_task_output, post_task_return_code = execute_entity_post_task_script(command, args.config, node_response.status_code, node_response.text)
+                post_task_output, post_task_return_code = execute_entity_post_task_script(command, config, node_response.status_code, node_response.text)
                 if post_task_return_code == 0:
```

Comment:
`args.config` is intentional here - it's the name of the current Workbench config file.

Reply:
I checked again, and of course I may be wrong, but this `args` variable in question is not defined or in scope in line 5617 of `workbench_utils.py` file... 

`post_task_output, post_task_return_code = execute_entity_post_task_script(command, args.config, node_response.status_code, node_response.text)`

or in its enclosing function `create_children_from_directory()`, instead this function signature  accepts a `config` param
https://github.com/mjordan/islandora_workbench/blob/c3fe7e13e1ac16cd5a731555edaf376639b5513a/workbench_utils.py#L5468

here is where `create_children_from_directory()` is called from the `workbench` file with the `config` param set, 
https://github.com/mjordan/islandora_workbench/blob/c3fe7e13e1ac16cd5a731555edaf376639b5513a/workbench#L403-L405

of course in the `workbench` file the `args` variable is defined and in global scope, but it is not in scope within the `workbench_utils.py` file and its code.

Again, I probably would never have noticed this without using the PyCharm linter, but I may still be wrong. 

If you still prefer only the other line change, I will update the PR to only have the other change for you to merge.

Reply:
Yes, I would like to update the PR with your suggested fix. I will put in a new commit in my PR today.